### ISSUE TITLE
How to add base URL for this project?

### ISSUE THREAD

**Issue by darshit7** [adlnet/xapi-launch#17]
Is there any configurations to add base url for this application?

**Comment by FlorianTolk** [comment]
Good Morning @darshit7, is this what you were looking for?
```
//Sets up the wrapper
        var wrapper = ADL.XAPIWrapper;
        wrapper.changeConfig({
            "endpoint": "https://lrs.adlnet.gov/xapi/",
            "auth": "Basic " + toBase64('user:password')
        });

//baseURI is a URL that you can use later to add extensions to your JSON File
wrapper.baseURI = "http://example.com/";
```

**Comment by darshit7** [comment]
No. When I run xapi-launch server using command `node app`, it listen on `localhost:3000` but what I want is to make it available on `localhost:3000/<base_url>/`
All requests should be served on `localhost:3000/<base_url>`

**Comment by vbhayden** [comment]
Have you tried just modifying the `app.js` file to do that redirect logic for you?

**Comment by darshit7** [comment]
Actually, I am new to nodejs and don't know how to write logic for redirect. I would appreciate if you guide me. 
I have little bit idea about express Router but I don't know from where to start.
